assets/css/index.styl

```diff
@@ -35,6 +35,13 @@ h1
 h2
   font-size: 1.5em
   margin: 0.83em 0
+  font-size: 22px;
+  line-height: 22px;
+  letter-spacing: 4px;
+  margin-bottom: 10px;
+  font-weight: 400;
+  text-transform: uppercase;
+  text-align: center;
```

Comment:
Good eye. I should have specified the H2's.  it doesn't work on this page. Do you think that the larger font competes with the logo or that it works.